src/components/Card/__tests__/Card.test.tsx

```diff
@@ -12,28 +12,37 @@ jest.mock('react-router-dom', () => ({
 
 describe('Card', () => {
     it('should render card with single column', () => {
-        var columns = [{key: 'columnKey', value: 'columnValue'}];
-        render(<Card columns={columns} />);
+        const id = 'test-id';
+        const columns = [{key: 'columnKey', value: 'columnValue'}];
+        render(<Card id={id} columns={columns} />);
+
+        expect(screen.queryAllByTestId("card-column").length).toBe(columns.length);
```

Comment:
Added test to check if the number of columns in the created card is the same as the length of the columns passed to the component